cmd/config-current.go

```diff
@@ -43,7 +43,7 @@ import (
 // Config version
 const serverConfigVersion = "27"
 
-type serverConfig = serverConfigV27
+type serverConfig serverConfigV27
```

Comment:
@kannappanr I think type aliasing is better here because declaring new type won't inherit serverConfigV27 methods. Is there any reason for doing this change ?

Reply:
`make test` fails on my machine without this change

Reply:
> make test fails on my machine without this change

It is better to know why it fails, does it show any error message ?

Reply:
```
 make test
Installing golint
Installing gocyclo
Installing deadcode
Installing misspell
Installing ineffassign
Running vet
Running fmt
cmd/config-current.go:46:19: expected type, found '='
cmd/format-fs.go:50:17: expected type, found '='
Makefile:30: recipe for target 'fmt' failed
make: *** [fmt] Error 2
```
more specifically

```
$ gofmt -d cmd
cmd/config-current.go:46:19: expected type, found '='
cmd/format-fs.go:50:17: expected type, found '='
```

Reply:
This looks like your local issue @kannappanr https://github.com/golang/go/issues/21645

Reply:
https://github.com/golang/go/issues/25430 also why this is not readily available @kannappanr

Reply:
Yes, it is. Fixed it.